modules/pam_env/pam_env.8.xml

```diff
@@ -52,13 +52,40 @@
       variables as well as <emphasis>PAM_ITEM</emphasis>s such as
       <emphasis>PAM_RHOST</emphasis>.
     </para>
-    <para>
+    <para condition="with_vendordir">
+      Rules for (un)setting of variables can be defined in an own conig
+      file. The path of this file can be specified with the
+      <emphasis>conffile</emphasis> option.
+      If this file has not been defined the default rules are taken from the
+      config files <filename>/etc/security/pam_env.conf</filename> and
+      <filename>/etc/security/pam_env.conf.d/*.conf</filename>.
+      If the file <filename>/etc/security/pam_env.conf</filename> is not
+      available the rules are taken from the files
+      <filename>/usr/etc/security/pam_env.conf</filename>,
+      <filename>/usr/etc/security/pam_env.conf.d/*.conf</filename> and
+      <filename>/etc/security/pam_env.conf.d/*.conf</filename> in that order.
+    </para>
+    <para condition="without_vendordir">
       By default rules for (un)setting of variables are taken from the
       config file <filename>/etc/security/pam_env.conf</filename>. An
       alternate file can be specified with the <emphasis>conffile</emphasis>
       option.
     </para>
-    <para>
+    <para condition="with_vendordir">
+      Environment variables can be defined in file with simple <emphasis>KEY=VAL</emphasis>
+      pairs on separate lines. The path of this file can be specified with the
+      <emphasis>envfile</emphasis> option.
+      If this file has not been defined the settings are taken from the
+      files <filename>/etc/security/environment</filename> and
+      <filename>/etc/security/environment.d/*</filename>.
+      If the file <filename>/etc/environment</filename> is not available the
+      settings are taken from the files <filename>/usr/etc/environment</filename>,
+      <filename>/usr/etc/environment.d/*</filename> and
+      <filename>/etc/environment.d/*</filename> in that order.
+      And last but not least with the <emphasis>readenv</emphasis> option this can
+      be completely disabled.
+    </para>
+    <para condition="without_vendordir">
```

Comment:
This creates an impression that all these new files are supported when linux-pam is configured using `--enable-vendordir`, but the implementation does something completely different: the only place where `VENDORDIR` is going to be used is a new function `_econf_read_file` which is naturally under `USE_ECONF` guard.

In other words, this PR is not about `--enable-vendordir` support in pam_env but rather about `--enable-econf` support in pam_env, and about new features supported only if linux-pam is configured using `--enable-econf`.

I don't see why `--enable-econf` should be a prerequisite for `--enable-vendordir`, it certainly hasn't been discussed.